drltrace/scripts/gen_drltrace_config.py

```diff
@@ -353,10 +353,272 @@
 "LSTATUS":"LSTATUS",
 "DEP_SYSTEM_POLICY_TYPE":"DEP_SYSTEM_POLICY_TYPE",
 "CURRENCYFMTA":"CURRENCYFMTA",
-"va_list":"va_list"}
+"va_list":"va_list",
```

Comment:
Would it make sense to omit those that just identity mappings and use the original name for missing entries?  That would make this list much shorter.  Having an automated rule to remove a leading P is a little harder b/c some type names start with P.

Reply:
Yes, it makes sense.

Reply:
There is a problem with this idea, if I understood you right. I can not just remove mappings e.g. "LSTATUS":"LSTATUS because I usually manually analyze all new types found in the SDK files to indentify pointers. For example, if I remove mappings, it would be hard for me to understand which type I already saw and which one is new. Potentially, we can have a list of already parsed types, it will also make this map shorter (but it will increase search time).

Reply:
Yes, makes sense that this serves as a list of known types.